sources/packages/web/src/components/aest/institution/modals/ApproveProgramModal.vue

```diff
@@ -74,6 +74,8 @@ export default {
     },
   },
   setup() {
+    const NOTES_MAX_CHARACTERS = 500;
```

Comment:
Good catch about validating the length here, it was done by form.io before.
I would say that this will be the same for all the note validations. Having the generic method `checkMaxCharacters` is fine but maybe we can have a `notesValidator` centralized in the composables to be reused among all note forms.